packages/util/src/string-formatter.ts

```diff
@@ -0,0 +1,43 @@
+import voca from 'voca';
+
+/**
+ * Convert a string to the uppercase snake case. This format is used e. g. for static properties on entity classes.
+ *
+ * @param str - The string to be transformed.
+ * @returns The input string in the case used by static methods on entity-classes.
+ */
+export function toUpperCaseSnakeCase(str: string): string {
+  return voca.upperCase(voca.snakeCase(str));
+}
+
+/**
+ * Convert a string to camelCase. This format used e. g. for properties on entity class instances.
+ *
+ * @param str - The string to be transformed.
+ * @returns The transformed string.
+ */
+export function toCamelCase(str: string): string {
+  return voca.camelCase(str);
+}
+
+/**
+ * Convert a string to a human readable format, e.g. it transforms `to_BusinessPartner` to `To Business Partner`.
+ *
+ * @param str - The string to be transformed.
+ * @returns The transformed string.
+ */
+export function toTitleFormat(str: string): string {
```

Comment:
This function has a slightly different name than the others. The reason is, that we might want to add a `toTitleCase` function that transforms `content-type` to `Content-Type` as well.

Reply:
fine for me. `Format` is a good word since it is not changing the case.